startup/GafferCyclesUI/metadata.py

```diff
@@ -0,0 +1,592 @@
+        "color" : 1,
+        "color_attribute" : 2,
+        "density" : 3,
+        "density_attribute" : 4,
+        "anisotropy" : 5,
+        "absorption_color" : 6,
+        "emission_strength" : 7,
+        "emission_color" : 8,
+        "blackbody_intensity" : 9,
+        "blackbody_tint" : 10,
+        "temperature" : 11,
+        "temperature_attribute" : 12,
+    },
+    "principled_hair_bsdf" : {
+        "color" : 1,
+        "absorption_coefficient" : 2,
+        "melanin" : 3,
+        "melanin_redness" : 4,
+        "tint" : 5,
+        "roughness" : 6,
+        "radial_roughness" : 7,
+        "coat" : 8,
+        "ior" : 9,
+        "offset" : 10,
+        "random_roughness" : 11,
+        "random_color" : 12,
+        "random" : 13,
+        "R" : 14,
+        "TT" : 15,
+        "TRT" : 16,
+        "aspect_ratio" : 17,
+    },
+    "bump" : {
+        "invert" : 9,
+        "strength" : 1,
+        "distance" : 2,
+        "height" : 3,
+        "normal" : 4,
+        "sample_center" : 5,
+        "sample_x" : 6,
+        "sample_y" : 7,
+        "use_object_space" : 9,
+    },
+    "ambient_occlusion" : {
+        "samples" : 1,
+        "inside" : 2,
+        "only_local" : 3,
+        "color" : 4,
+        "distance" : 5,
+        "normal" : 6,
+    },
+    "glass_bsdf" : {
+        "distribution" : 1,
+        "color" : 2,
+        "roughness" : 3,
+        "ior" : 4,
+        "normal" : 5,
+    },
+    "hair_bsdf" : {
+        "component" : 1,
+        "color" : 2,
+        "roughness_u" : 3,
+        "roughness_v" : 4,
+        "offset" : 5,
+        "tangent" : 6,
+    },
+    "toon_bsdf" : {
+        "component" : 1,
+        "color" : 2,
+        "size" : 3,
+        "smooth" : 4,
+        "normal" : 5,
+    },
+    "glossy_bsdf" : {
+        "distribution" : 1,
+        "color" : 2,
+        "roughness" : 3,
+        "anisotropy" : 4,
+        "rotation" : 5,
+        "tangent" : 6,
+        "normal" : 7,
+    },
+    "refraction_bsdf" : {
+        "distribution" : 1,
+        "color" : 2,
+        "roughness" : 3,
+        "ior" : 4,
+        "normal" : 5,
+    },
+    "sheen_bsdf" : {
+        "distribution" : 1,
+        "color" : 2,
+        "roughness" : 3,
+        "normal" : 4,
+    },
+    "diffuse_bsdf" : {
+        "color" : 1,
+        "roughness" : 2,
+        "normal" : 3,
+    },
+    "mix" : {
+        "mix_type" : 1,
+        "factor" : 2,
+        "color1" : 3,
+        "color2" : 4,
+        "use_clamp" : 5,
+    },
+    "subsurface_scattering" : {
+        "method" : 1,
+        "color" : 2,
+        "radius" : 3,
+        "scale" : 4,
+        "subsurface_ior" : 5,
+        "subsurface_anisotropy" : 6,
+        "normal" : 7,
+    },
+    "math" : {
+        "math_type" : 1,
+        "value1" : 2,
+        "value2" : 3,
+        "value3" : 4,
+        "use_clamp" : 5,
+    },
+
+}
+
+defaults = {
+    "principled_bsdf" : {
+        #"subsurface_radius.x" : 1, why doesn't this work?
```

Comment:
I think `"subsurface_radius" : imath.Color3f( 1, .2, .1 )` might be what you're after here.

Reply:
That sounds right. The metadata is registered against `parameters.*` which only applies to the immediate children of `parameters`. There is a `parameters...*` syntax you could use to match recursively, but I think @boberfly's suggestion is better.